scripts/run_boss_locally.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+
+import os
+import sys
+import tempfile
+import shutil
+import glob
+import pydbus
+import shutil
+from gi.repository import Gio
+
+os.putenv("PYTHONPATH", os.path.abspath(".."))
+
+MODULES_DIR = os.path.abspath("../pyanaconda/modules")
+DBUS_SERVICES_DIR = "../data/dbus/"
+
+print("creating a temporary directory for DBUS service files")
+temp_service_dir = tempfile.TemporaryDirectory()
```

Comment:
It would be nice to have a prefix for the temp dir.

Reply:
While I don't find such thing very useful it looks like a prefix can be added for a temporary directory. :)